Gems/Atom/Feature/Common/Assets/Materials/Types/NewPBR/NewPBR.azsli

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+
+// ------- Options -------
+
+#define ENABLE_CLEAR_COAT       0
```

Comment:
A word about this choice of macro flags and where and how they are used would be a good add.  Is this something that will not be overridden by the material option?